### ISSUE TITLE
empty blocks clock skew with IBC

### ISSUE THREAD

**Issue by mtsitrin** [dymensionxyz/dymint#469]
```
46245244-1", "attempt": 1, "max_attempts": 30, "error": "rpc error: code = Unknown desc = rpc error: code = Unknown desc = failed to execute message; message index: 0: cannot update client with ID 07-tendermint-0: failed to verify header: invalid header: new header has a time from the future 2023-08-27 17:03:25.377770692 +0000 UTC (now: 2023-08-27 16:48:07.152709 +0000 UTC; max clock drift: 10m0s) [dymensionxyz/ibc-go/v6@v6.0.0-rc0.0.20230619115257-1a1038e16d71/modules/light-clients/07-tendermint/types/update.go:245] With gas wanted: '18446744073709551615' and gas used: '158869' : unknown request"}
```

**Comment by mtsitrin** [comment]
when a rollapp handles hub's `updateClient`, it checks
the received header's timestamp, against rollapp's last header timestamp (allowing maxDrift which is 10mins)


with empty blocks, the rollapp's header isn't updated frequently, those the timestamp isn't updated.


I suggest to increase the 
`MaxClockDrift:   time.Minute * 10,` in the relayer code
to `70 mins (emptyBlocks default time + 10 Mins)`

@omritoptix WDYT

**Comment by yishay-dym** [comment]
why is this timestamp interval verified and enforced? 
What other parts rely or are effected by this "drift"?

**Comment by mtsitrin** [comment]
blocked by https://github.com/dymensionxyz/research/issues/172

**Comment by mtsitrin** [comment]
closed by #471
